light-client/src/predicates.rs

```diff
@@ -261,6 +261,13 @@ pub fn verify(
         &trusted.signed_header.header,
     )?;
 
+    // Verify that more than 2/3 of the validators correctly committed the block.
```

Comment:
I believe this is false security and uneconomical - it's trivial for a faulty node to make this pass now even though we might never be able to extend trust to this header in the first place. I think it also means we end up doing this verification twice in the light node? 

The performance issue isn't that big a deal to the light-node, but for IBC handlers it probably doesn't make sense to do this more expensive check before the +1/3 check.

Reply:
I still think we should revert this. It creates DoS attacks for light clients since full nodes can now spam us with massive commits (eg. 10k validators...) https://github.com/tendermint/tendermint/pull/5606

I guess it means we need to fix the MBT tests though to better describe why certain cases should fail?

Reply:
Ok, will open a new PR to revert this. We can perhaps do the required changes in MBT tests as part of that PR as well.